src/components/styles/form.css

```diff
@@ -0,0 +1,41 @@
+.form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+form {
+  display: flex;
+  gap: 20px;
+}
+
+.form-title {
+  font-size: 1.17rem;
+  color: rgb(136, 136, 136);
+}
+
+form > input {
+  width: 50%;
+  border: none;
+  padding: 16px;
+  font-size: 17px;
+}
+
+form > select {
+  width: 30%;
+  border: none;
+  padding: 16px;
+  color: grey;
+}
```

Comment:
- [ ] In this task we are to follow the provided design as strictly as possible and that includes colors, font faces, typography, e.t.c. Please implement the design for this input field and the select field as in the original design. The screenshot below will help show us the expectation:

  | The Design | Your implementation |
  | --- | --- |
  | ![image](https://user-images.githubusercontent.com/45002654/155512906-bf1293d3-31e5-442c-8405-eaa19254e519.png) | ![image](https://user-images.githubusercontent.com/45002654/155512939-2e7844e0-ede6-4b18-aabf-abeb5a850e3d.png) |

Reply:
- this is how it looks on my screen,I think it matches the design


![image](https://user-images.githubusercontent.com/90524466/155527248-30d01fe8-1f1a-4c52-99bf-8a91dac1869b.png)

Reply:
I think you are good to go. I have realized pages are rendering with anomalies on my browser today. Sorry for this.